src/backend/tests/test-data/projects.test-data.ts

```diff
@@ -33,3 +35,49 @@ export const someProject = {
     ]
   }
 };
+
+export const createWorkPackagePayload = {
```

Comment:
the payloads can stay in the test files

Reply:
actually putting this in a data file isn't that bad, but it hsould be in a new file `work-packages.test-data.ts`